test/fluree/db/query/federated_test.clj

```diff
@@ -0,0 +1,285 @@
+(ns fluree.db.query.federated-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [fluree.db.test-utils :as test-utils]
+            [fluree.db.json-ld.api :as fluree]))
+
+(deftest ^:integration federated-query-connection-test
+  (testing "Federated queries using query-connection"
+    (let [conn    (test-utils/create-conn)
+          context {"id"     "@id",
+                   "type"   "@type",
+                   "ex"     "http://example.org/",
+                   "f"      "https://ns.flur.ee/ledger#",
+                   "rdf"    "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+                   "rdfs"   "http://www.w3.org/2000/01/rdf-schema#",
+                   "schema" "http://schema.org/",
+                   "xsd"    "http://www.w3.org/2001/XMLSchema#"}
+
+          authors @(fluree/create-with-txn conn
+                                           {"@context" ["https://ns.flur.ee" context "https://schema.org"]
+                                            "ledger"   "test/authors"
+                                            "insert"   [{"@id"   "https://www.wikidata.org/wiki/Q42"
+                                                         "@type" "Person"
+                                                         "name"  "Douglas Adams"}
+                                                        {"@id"   "https://www.wikidata.org/wiki/Q173540"
+                                                         "@type" "Person"
+                                                         "name"  "Margaret Mitchell"}]})
+          books   @(fluree/create-with-txn conn
+                                           {"@context" ["https://ns.flur.ee" context "https://schema.org"]
+                                            "ledger"   "test/books"
+                                            "insert"   [{"id"     "https://www.wikidata.org/wiki/Q3107329",
+                                                         "type"   ["Book"],
+                                                         "name"   "The Hitchhiker's Guide to the Galaxy",
+                                                         "isbn"   "0-330-25864-8",
+                                                         "author" {"@id" "https://www.wikidata.org/wiki/Q42"}}
```

Comment:
Ok, I'm really late to this, but I just noticed that the iri here ends with "Q42" :)